martin-mbtiles/src/bin/main.rs

```diff
@@ -69,20 +58,8 @@ async fn main() -> Result<()> {
         Commands::MetaGetValue { file, key } => {
             meta_get_value(file.as_path(), &key).await?;
         }
-        Commands::Copy {
-            src_file,
-            dst_file,
-            min_zoom,
-            max_zoom,
-            zoom_levels,
-        } => {
-            let copy_opts = TileCopierOptions::new()
-                .verbose(args.verbose)
-                .min_zoom(min_zoom)
-                .max_zoom(max_zoom)
-                .zooms(zoom_levels);
-
-            let tile_copier = TileCopier::new(src_file, dst_file, copy_opts)?;
+        Commands::Copy { opts } => {
+            let tile_copier = TileCopier::new(opts)?;
```

Comment:
I think the `TileCopier` should do the copying while the `TileCopierOptions` should just hold the options and do nothing else. The `TileCopier` object is what is being exposed to the user.

Reply:
Also, I think it makes more sense conceptually to create a `TileCopier` with options and _then_ have it `run` and do the copying as opposed to doing it all at once. If we did it all at once, it would seem like the struct is redundant and we could do it in a single function. The `new` function serves to construct the internal `src_mbtiles` struct so I'd prefer to leave it as is.

Reply:
Per discussion - you can simplify it with 

```rust
    #[command(name = "copy")]
    Copy(TileCopierOptions),
```